aio-subpath-access.Caddyfile

```diff
@@ -0,0 +1,37 @@
+:3000 {
+	respond 404
+}
+
+:3100 {
+	respond 404
+}
+
+:3170 {
+	reverse_proxy localhost:8080
+}
```

Comment:
The callback URLs remain the same throughout, which is on port `3170`. This ensures it is accessible.